File: src/dartCode/max_wealth.dart
void main() {
  var accounts = [
    [1, 2, 3],
    [3, 2, 1]
  ];
  print(maximumWealth(accounts));
}

int maximumWealth(List<List<int>> accounts) {
  int ans = 0;
  for (List<int> ints in accounts) {
    int sum = 0;
    for (int anInt in ints) {
      sum += anInt;
    }
    if (sum > ans) {
      ans = sum;
    }
  }
  return ans;
}

File: src/kotlinCode/MaxWealth.kt
package kotlinCode

fun main() {
    val accounts = arrayOf(intArrayOf(1, 2, 3), intArrayOf(3, 2, 1))
    println(maxWealth(accounts))
}

fun maxWealth(accounts: Array<IntArray>): Int {
    var ans = Integer.MIN_VALUE
    for (ints: IntArray in accounts) {
        var sum = 0
        for (anInt: Int in ints) {
            sum += anInt
        }
        if (sum > ans) {
            ans = sum;
        }
    }
    return ans
}